specification/compute/resource-manager/Microsoft.Compute/preview/2020-09-30/sharedGallery.json

```diff
@@ -0,0 +1,1040 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SharedGalleryServiceClient",
+    "description": "Shared Gallery Service Client.",
+    "version": "2020-09-30"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/sharedGalleries": {
```

Comment:
is there not an API to list all sharedGalleries regardless of location?  You could use Fanout routing

Reply:
That would be enormous. Currently I think we only want to support list all shared galleries in one location. Cause a lot of subscriptions are location-based too.